CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Autogenerated by auto2cmake on 2018-03-07 11:49:59
+cmake_minimum_required(VERSION 3.9)
```

Comment:
please support 3.7.2 (available in debian stretch)
I tested with 3.7.2 and it works